spec/models/category_list_spec.rb

```diff
@@ -387,5 +387,27 @@ def ordered_category_list(some_user)
         category_list.categories.find { |c| c.id == category.id }.subcategory_ids,
       ).to contain_exactly(subcategory.id)
     end
+
+    it "returns at most SUBCATEGORIES_PER_CATEGORY subcategories" do
+      Fabricate(:category, user: admin, parent_category: category)
+      Fabricate(:category, user: admin, parent_category: category)
+
+      category_list =
+        stub_const(CategoryList, "SUBCATEGORIES_PER_CATEGORY", 2) do
```

Comment:
```suggestion
        stub_const(CategoryList, "SUBCATEGORIES_PER_CATEGORY", 1) do
```

We only created two child categories so setting the const to 1 doesn't demonstrate that we are restricting the number of subcategories being loaded.

Reply:
There is another subcategory being created at a higher level (see line 380), but I think I can reduce `SUBCATEGORIES_PER_CATEGORY` by 1 anyway.

Reply:
I updated this and made the code slightly more readable (hopefully).